project/application/src/components/reportsSearchBar/index.js

```diff
@@ -323,11 +317,14 @@ export default class ReportsSearchBar extends Component {
         client.request(getUsers()).then(data => {
             this.setState({ selectUersData: data.user });
             this.setState({ selectUersDataEtalon: data.user });
+            this.setState({ selectUserData: this.props.setUser})
```

Comment:
what do you want to do via this line? I see that selectUserData key already updated in selectUserData at line 318